src/platform/utilities/feature-toggles/flipperID.js

```diff
@@ -0,0 +1,28 @@
+const Cookies = require('js-cookie');
+
+// Read GA Id if available otherwise use a randomString as a token
+function getToken() {
+  const randomString =
+    Math.random()
+      .toString(36)
+      .substring(2, 15) +
+    Math.random()
+      .toString(36)
+      .substring(2, 15);
+
+  return Cookies.get('_vagovRollup_gid')
+    ? Cookies.get('_vagovRollup_gid')
+    : randomString;
+}
+
+// Create cookie containing token
+function setFlipperCookie() {
+  if (!Cookies.get('FLIPPER_ID')) {
+    Cookies.set('FLIPPER_ID', getToken(), { expires: 30 });
+  }
+  return getToken();
```

Comment:
It looks like we could set the `FLIPPER_ID` to a random string, and then return a completely different random string from this function. That's not what we want to do, right?